engine/src/conversion/analysis/fun/mod.rs

```diff
@@ -1675,6 +1768,16 @@ impl<'a> FnAnalyzer<'a> {
                     },
                 )
             }
+            if items_found.implicit_destructor_needed() {
+                self.synthesize_constructor(
```

Comment:
Could we rename this to `synthesize_special_member`?

Reply:
Done